### PR TITLE
feat: new dashboards page [sc-14069]

### PR DESCRIPTION
**Pull request by pilimartinez**
## Affected Components
* [X] Content & Marketing
* [ ] Pricing
* [ ] Test
* [ ] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer

New dashboards page under product/dashboards

### CODE REVIEW

File: site/content/product/dashboards.mmark [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+<div class="mt-4 pb-9">
+  <div class="row justify-content-center header-part">
+    <div class="col-sm-12 col-md-8">
+      <h2 class="text-center section-header mt-1">How it works</h3>
+      <p class="text-center hero-sub hero-sub-center mb-3">
+        In addition to being fast and responsive, our dashboards are designed with readability and accessibility in mind.
+      </p>
+      <p class="lead-text text-center mb-5">
+        {{% cta-link "Try the New Checkly Dashboards Today" "https://app.checklyhq.com/signup" %}}
```
Comment (tnolet) [inline site/content/product/dashboards.mmark:39]:
```suggestion
        {{% cta-link "Try the new Checkly Dashboards today" "https://app.checklyhq.com/signup" %}}
```

File: site/content/product/dashboards.mmark [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    </div>
+  </div>
+</div>
+
+<div class="gray-section">
+  <div class="row justify-content-center header-part">
+    <div class="col-sm-12 col-md-8">
+      <h2 class="section-header mt-1">Built for Speed</h2>
+      <p class="lead-text mb-lg-2">
+      Our new dashboards are built on NuxtJS 3 and leverage NuxtJS Universal Rendering capabilities. We host the new dashboard on Vercel, and use the Vercel Domains API to properly handle custom domains. The result? Jaw-dropping-fast dashboard loads and refreshes.
```
Comment (tnolet) [inline site/content/product/dashboards.mmark:81]:
```suggestion
      Our new dashboards are built on NuxtJS 3 and leverage NuxtJS Universal Rendering capabilities. We host the new dashboards on Vercel, and use the Vercel Domains API to properly handle custom domains. The result? Jaw-dropping-fast dashboard loads and refreshes. 
```

**Review by tnolet — CHANGES_REQUESTED** [review]
Some tiny copy changes I missed earlier

**Review by tnolet — APPROVED** [review]
LGTM!

**Review by shiini2 — APPROVED** [review]
LGTM! 👍

**Review by ndom91 — APPROVED** [review]

**Pull request merged**